server/apps/distributed-app/docs/modules/ROOT/pages/configure/extensions.adoc

```diff
@@ -25,4 +25,34 @@ guice.extension.module=com.project.MyServiceModule
 
 Enables to inject MyService into your extensions.
 
+
+*guice.extension.module*: come separated list of fully qualified class name. These classes need to implement Task extension modules.
+
+Here is an example of such a class :
+
+....
+public class RSpamDTaskExtensionModule implements TaskExtensionModule {
+
+    @Inject
+    public RSpamDTaskExtensionModule() {
+    }
```

Comment:
Why do we need this?

Reply:
just a sample of the document

the class should have Inject annotation with constructor for guice